library/options.inc.php

```diff
@@ -2036,7 +2036,7 @@ function generate_display_field($frow, $currvalue)
         }
         // If match is not found in main and backup lists, return the key with exclamation mark
         if ($s == '') {
-            $s = nl2br(htmlspecialchars($currvalue, ENT_NOQUOTES)).
+            $s = nl2br(text(xl_list_label($currvalue), ENT_NOQUOTES)).
```

Comment:
so close :) (need to remove the `, ENT_NOQUOTES`)
```
$s = nl2br(text(xl_list_label($currvalue))).
```

Reply:
oops..
Done